src/common/arrow/src/native/stat.rs

```diff
@@ -209,7 +209,7 @@ mod test {
             array.validity().is_some(),
         );
         let schema = Schema::from(vec![field.clone()]);
-        let mut writer = NativeWriter::new(&mut bytes, schema, options);
+        let mut writer = NativeWriter::new(&mut bytes, schema, options).unwrap();
```

Comment:
Is it safe to unwrap? invalid data may cause a panic.

Reply:
it's in test mod

Reply:
ok